### PR TITLE
remove outdated :initial-props

### PR DESCRIPTION
**Pull request by flexsurfer**
remove outdated :initial-props

### CODE REVIEW

File: src/status_im/node/core.cljs [hunk 1]
```diff
@@ -50,15 +50,11 @@
       les-topic (assoc les-topic {:Min 2, :Max 2}))))
 
 (defn- get-base-node-config [config]
-  (let [initial-props @(re-frame/subscribe [:initial-props])
-        status-node-port (get initial-props :STATUS_NODE_PORT)]
```
Comment (flexsurfer) [inline src/status_im/node/core.cljs:54]: @siphiuel , git says you've added these lines, do you remember why ?

File: src/status_im/node/core.cljs [hunk 2]
```diff
@@ -50,15 +50,11 @@
...
-    (cond-> (assoc config
-                   :Name "StatusIM")
-      config/dev-build?
-      (assoc :ListenAddr ":30304"
-             :DataDir (str (:DataDir config) "_dev"))
-      status-node-port
-      (assoc :ListenAddr (str ":" status-node-port)))))
```
Comment (flexsurfer) [inline src/status_im/node/core.cljs:61]: @cammellos do you know what is `:ListenAddr` ?

Comment (cammellos) [reply src/status_im/node/core.cljs:61]: It's the port the client binds, on desktop you probably wanted to run multiple instances, so you passed it as an ENV variable

**Comment by flexsurfer** [comment]
@cammellos could you approve pls

**Review by cammellos — APPROVED** [review]
@flexsurfer sorry, I thought I did the first time :facepalm:

**Pull request merged**